<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云镝AI智能视觉检测系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Kingdee_ico.ico</normaloff>Kingdee_ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>相机实时检测区（实时拍照+结果显示）</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数据实时分析区   （正太分布图）（3西格玛）</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="7,3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
            <item>
             <widget class="QGraphicsView" name="graphicsView"/>
            </item>
            <item>
             <widget class="QGraphicsView" name="date_view"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;显示报错信息和执行信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="dateTime">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>日期显示</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;此处主要用于显示检测的数据信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件（F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑（E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_X">
    <property name="title">
     <string>执行（X)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>可视化（V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>函数（P)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_D">
    <property name="title">
     <string>算子（D)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>建议(S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>助手（A)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口（W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（H）</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_X"/>
   <addaction name="menu_V"/>
   <addaction name="menu_P"/>
   <addaction name="menu_D"/>
   <addaction name="menu_S"/>
   <addaction name="menu_A"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭项目</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存所有</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
